### PR TITLE
Huge Traits Rename*

### PR DESCRIPTION
**Pull request by KirbyDaMaster**
## About The Pull Request
Renamed most traits in _defines/traits.dm to make file uniform and easy to understand, as well as cleaning up the overall appearance of the file. 

There is a new precedent for trait/source naming conventions, all traits begin with TRAIT_ and all sources begin with SOURCE_. This was determined to be the clearest solution as well as the tidiest. 

## Why It's Good For The Game
Because clean clean sweep sweep. We all love code that we can read. 

## Changelog

:cl:
rename: renamed many traits for clarity in traits.dm along with all associated references.
/:cl:

### CODE REVIEW

File: code/__DEFINES/traits.dm [hunk 1]
```diff
@@ -118,321 +118,418 @@
...
+//Inability to access UI hud elements. Turned into a trait from [MOBILITY_UI] to be able to track sources.
 #define TRAIT_UI_BLOCKED "uiblocked"
-/// Inability to pull things. Turned into a trait from [MOBILITY_PULL] to be able to track sources.
+
+//Inability to pull things. Turned into a trait from [MOBILITY_PULL] to be able to track sources.
 #define TRAIT_PULL_BLOCKED "pullblocked"
-/// Abstract condition that prevents movement if being pulled and might be resisted against. Handcuffs and straight jackets, basically.
+
+//Abstract condition that prevents movement if being pulled and might be resisted against. Handcuffs and straight jackets, basically.
 #define TRAIT_RESTRAINED "restrained"
```
Comment (LemonInTheDark) [inline code/__DEFINES/traits.dm:145]: Why the newlines?

File: code/__DEFINES/traits.dm [hunk 2]
```diff
@@ -863,68 +894,195 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
...
+//Ignore Crew monitor Z levels
 #define TRAIT_MULTIZ_SUIT_SENSORS "multiz_suit_sensors"
 
-/// Ignores body_parts_covered during the add_fingerprint() proc. Works both on the person and the item in the glove slot.
+//Ignores body_parts_covered during the add_fingerprint() proc. Works both on the person and the item in the glove slot.
 #define TRAIT_FINGERPRINT_PASSTHROUGH "fingerprint_passthrough"
 
-/// this object has been frozen
+//this object has been frozen
 #define TRAIT_FROZEN "frozen"
```
Comment (LemonInTheDark) [inline code/__DEFINES/traits.dm:1076]: ?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
Also I noticed you renamed a few things, what were they exactly?

**Comment by KirbyDaMaster** [comment]
> Also I noticed you renamed a few things, what were they exactly?

I am not sure anymore, I burnt myself out and needed a bit to breathe.

**Pull request closed without merging**